File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/backend/response/DownloadMyAvailableBordersResponseContent.java
package de.dimedis.mobileentry.backend.response;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class DownloadMyAvailableBordersResponseContent extends BaseResponseContent {
    public static final String BORDERS = "borders";

    @SerializedName(BORDERS)
    private List<Border> listBorders;

    public List<Border> getListBorders() {
        return listBorders;
    }

    @Override
    public String toString() {
        return "DownloadMyAvailableBordersResponseContent{" +
                "listBorders=" + listBorders +
                '}';
    }
}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/backend/response/DownloadMyAvailableBordersResponseContent.kt
package de.dimedis.mobileentry.backend.response

import com.google.gson.annotations.SerializedName

class DownloadMyAvailableBordersResponseContent : BaseResponseContent() {
    companion object {
        const val BORDERS = "borders"
    }

    @SerializedName(BORDERS)
    val listBorders: List<Border?>? = null

    override fun toString(): String {
        return "DownloadMyAvailableBordersResponseContent{" +
                "listBorders=" + listBorders +
                '}'
    }
}